core-lightning/docker-compose.yml

```diff
@@ -64,6 +64,7 @@ services:
       - --bitcoin-rpcconnect=${APP_BITCOIN_NODE_IP}
       - --bitcoin-rpcuser=${APP_BITCOIN_RPC_USER}
       - --bitcoin-rpcpassword=${APP_BITCOIN_RPC_PASS}
+      - --bitcoin-rpcport=8332
```

Comment:
We can use the environment variable `${$APP_BITCOIN_RPC_PORT}` here instead of `8332` because we export it from the [Bitcoin app](https://github.com/getumbrel/umbrel-apps/blob/a1579886cd0c366b81f6312ef5eba43c8b0180b5/bitcoin/exports.sh#L7). This way any changes we make to the Bitcoin app will automatically get carried over here.

Reply:
fixing